homeassistant/components/zwave.py

```diff
@@ -92,6 +93,10 @@
      [COMMAND_CLASS_DOOR_LOCK],
      TYPE_BOOL,
      GENRE_USER),
+    ('rollershutter',
+     [COMMAND_CLASS_SWITCH_MULTILEVEL],
```

Comment:
Yes, that list I am aware of, but the specific class is what I am interessted in. In this case MOTOR_CONTROL_CLASS_B is 6. Is there a list that those mappings are available?